lib/worker.sh

```diff
@@ -160,19 +160,19 @@ function LintCodebase() {
       ######################################################
       # Make sure we don't lint node modules or test cases #
       ######################################################
-      if [[ ${FILE} == *"node_modules"* ]]; then
+      if [[ ${DIR_NAME} == *"node_modules"* ]]; then
         # This is a node modules file
         continue
-      elif [[ ${FILE} == *"${TEST_CASE_FOLDER}"* ]]; then
+      elif [[ ${DIR_NAME} == *"${TEST_CASE_FOLDER}"* ]]; then
         # This is the test cases, we should always skip
         continue
-      elif [[ ${FILE} == *".git"* ]]; then
+      elif [[ ${DIR_NAME} == *".git"* ]] || [[ ${DIR_NAME} == *".git/"* ]]; then
```

Comment:
```suggestion
      elif [[ ${DIR_NAME} == *".git" ]] || [[ ${DIR_NAME} == *".git/"* ]]; then
```

`*".git"*` is include .github/...